holoviews/plotting/bokeh/chart.py

```diff
@@ -188,6 +188,8 @@ class VectorFieldPlot(ColorbarPlot):
         distance between vectors, this can be disabled with the
         rescale_lengths option.""")
 
+    padding = param.ClassSelector(default=0.05, class_=(int, float, tuple))
```

Comment:
`class_=(int, float, tuple))` is probably common enough in HoloViews code that it should be an actual parameter class, something like NumberOrTuple, which would let the Parameter implement checking that the tuple is numeric.  Here a non-numeric tuple would presumably not be caught until much later in the code, and there's no clear user-visible declaration that it would even need to be numeric.  Not something for this PR, necessarily, but in general it seems like HoloViews probably needs more custom Parameter types to achieve better error handling in cases like this.

Reply:
>HoloViews probably needs more custom Parameter types to achieve better error handling in cases like this.

Yes, it's a huge shame that HoloViews does not define any custom parameters...pretty embarrassing tbh given that it's almost certainly the largest param based project.

Reply:
Yup.